docker/build_docker.sh

```diff
@@ -159,6 +173,13 @@ function docker_build {
     cp Dockerfile "$tmp_dir/Dockerfile"
     echo "" >> "$tmp_dir/Dockerfile"
     echo "ENV DOCKER_IMAGE=$image_full_name" >> "$tmp_dir/Dockerfile"
+    
+    if [[ "$(prop 'deprecated')" ]]; then
+        echo "DEPRECATED_IMAGE=true" >> "$tmp_dir/Dockerfile"
```

Comment:
I think you need `ENV`

```suggestion
        echo "ENV DEPRECATED_IMAGE=true" >> "$tmp_dir/Dockerfile"
```